pkgs/applications/editors/molsketch/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, stdenv, mkDerivation, fetchgit, cmake, pkg-config, hicolor-icon-theme, openbabel, desktop-file-utils, qttranslations }:
```

This is a long line.
```suggestion
{ lib
, stdenv
, mkDerivation
, fetchgit
, cmake
, pkg-config
, hicolor-icon-theme
, openbabel
, desktop-file-utils
, qttranslations 
}:
```